### PR TITLE
About us

### PR DESCRIPTION
**Pull request by conradbielitz**
Includes translation fixes after major merge as well as content in the about us section

### CODE REVIEW

File: src/Pages/Home.tsx [hunk 1]
```diff
@@ -56,27 +82,44 @@ const Home: React.FC = () => {
...
+                <span key={i}>
+                  {line}
+                  <br />
+                </span>
+              ))}
           </p>
+
+          {/* Team members */}
+
+          <Row gutter={[16, 16]} justify="center">
```
Comment (mischakolbe) [inline src/Pages/Home.tsx:113]: This doesn't seem to adjust dynamically for mobile layout, but we can look at that in a next iteration. Maybe something for @emgerold ?

**Review by mischakolbe — APPROVED** [review]
Approving PR, since it improves the state of things and I don't see any actual issues.

File: src/Pages/Home.tsx [hunk 2]
```diff
@@ -56,27 +82,44 @@ const Home: React.FC = () => {
...
         >
-          <h3>Our vision and mission</h3>
+          <h3>{t('about.visionMissionTitle')}</h3>
           <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut rhoncus
-            interdum ligula, sit amet ullamcorper sapien ullamcorper ut. Vivamus
-            dapibus, urna nec tempor luctus, mauris risus efficitur justo, et
-            gravida metus ex id enim.
+            {t('about.visionMissionText')
+              .split('\n')
```
[the remark this answers is not in the document]
Comment (mAkeddar) [reply src/Pages/Home.tsx:91]: From old memories, it seems like the right way to do it for our first milestone. But I'm not aware of the newest changes on this lib!
I'll check if there's a nicer way to do :)

[the remark this answers is not in the document]
Comment (mischakolbe) [reply src/Pages/Home.tsx:91]: Cool by me. That's interesting!

**Pull request merged**